### ISSUE TITLE
[PATCH] fix spelling mistakes

### ISSUE THREAD

**Issue by gregoa** [PerlDancer/Dancer2#1132]
In Debian we are currently applying the following patch to Dancer2.
We thought you might be interested in it too.

    Description: fix spelling mistakes
    Origin: vendor
    Author: gregor herrmann <gregoa@debian.org>
    Last-Update: 2016-02-13
    

The patch is tracked in our Git repository at
https://anonscm.debian.org/cgit/pkg-perl/packages/libdancer2-perl.git/plain/debian/patches/spelling.patch

Thanks for considering,
  gregor herrmann,
  Debian Perl Group

**Comment by xsawyerx** [comment]
:+1:

**Comment by veryrusty** [comment]
:+1:

**Comment by veryrusty** [comment]
Hmmm.. Maybe its just Valentines Day blues; but that patch doesn't seem to apply cleanly against current head, or the v0.166001 tag.  #couldjustbeme.

**Comment by veryrusty** [comment]
"merged" in c7963d4. Thanks @gregoa !!

I couldn't get the patch to apply (nor find a commit to cherry-pick somewhere obvious), so I recreated the two character patch with attribution.

I really love the work that the Debian perl team do, but there must be _some_ way to make it easier for upstream projects to apply these patches in a way that doesn't adversely impact everyones workloads. (Which may well imply I need to learn more ;) ).

**Comment by gregoa** [comment]
On Sun, 14 Feb 2016 01:50:23 -0800, Russell Jenkins wrote:

> Hmmm.. Maybe its just Valentines Day blues; but that patch doesn't
> seem to apply cleanly against current head, or the v0.166001 tag.
> #couldjustbeme.

Now I got curious and tried as well, and indeed, if I make a fresh
clone of the Dancer2 repo and download our patch, I can't apply the
patch (neiter with `git apply` nor with good old `patch`).

Looking a bit further, my impression is that lib/Dancer2/Core/App.pm
in the v0.166001 tag is too different from the file I created the
patch against, i.e. the file from the released tarball at
https://cpan.metacpan.org/authors/id/X/XS/XSAWYERX/Dancer2-0.166001.tar.gz

Not sure what's the best point to tackle this issue.

Thanks for manually "applying" the patch!


Cheers,
gregor

-- 
 .''`.  Homepage https://info.comodo.priv.at/ - OpenPGP key 0xBB3A68018649AA06
 : :' : Debian GNU/Linux user, admin, and developer -  https://www.debian.org/
 `. `'  Member of VIBE!AT & SPI, fellow of the Free Software Foundation Europe
   `-   NP: Bruce Springsteen: Countin' on a Miracle
